### ISSUE TITLE
Warning due to use of deprecated default_features

### ISSUE THREAD

**Issue by andrewdavidmackenzie** [plotters-rs/plotters#598]
In my build I noticed this, so I thought I'd mention it:

```
cargo build --features "gui"
warning: `default_features` is deprecated in favor of `default-features` and will not work in the 2024 edition
(in the `plotters` dependency)
    Finished `dev` profile [unoptimized + debuginfo] target(s) in 0.36s
```

**Comment by Philippe-Cholet** [comment]
This should be closed thanks to #604.

**Comment by andrewdavidmackenzie** [comment]
I think still no published release including this change?

**Comment by AaronErhardt** [comment]
> I think still no published release including this change?

The last release is from September, the PR merged in July, so actually this is published by now.

**Comment by andrewdavidmackenzie** [comment]
crates.io has 0.3.7 as the latest release:
https://crates.io/crates/plotters
uploaded on September 8th.

Pr was merged on July 1st
https://github.com/plotters-rs/plotters/pull/604

my project resolves 0.3 to 0.3.7 as would be expected
https://github.com/andrewdavidmackenzie/pigg/blob/master/Cargo.lock#L4397

but I am still getting this warning in my builds (even after a cargo update)
```
warning: /Users/andrew/workspace/pigg/Cargo.toml: `default_features` is deprecated in favor of `default-features` and will not work in the 2024 edition
(in the `plotters` dependency)
```

So, I am not sure what is wrong, maybe the version with the PR was not the one uploaded to crates.io?

**Comment by andrewdavidmackenzie** [comment]
clean fixed it, so thanks!
